packages/functional-tests/tests/react-conversion/resetPassword.spec.ts

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+import { BaseTarget } from '../../lib/targets/base';
+import { EmailHeader, EmailType } from '../../lib/email';
+
+function getReactFeatureFlagUrl(target: BaseTarget, path: string) {
+  return `${target.contentServerUrl}${path}?showReactApp=true`;
+}
+
+const NEW_PASSWORD = 'notYourAveragePassW0Rd';
+
+test.describe('reset password', () => {
+  test.beforeEach(async ({}, { project }) => {
+    test.slow(project.name !== 'local', 'email delivery can be slow');
+  });
+
+  test('can reset password', async ({ page, target, credentials, context }) => {
+    await page.goto(getReactFeatureFlagUrl(target, '/reset_password'));
+
+    // Verify react page has been loaded
+    expect(await page.locator('#root').isVisible()).toBeTruthy();
```

Comment:
I believe these can just be `await page.locator('#root').isVisible()` and that we don't need to check with `expect`/`toBeTruthy`, these should fail if it doesn't become visible. I'd searched around for best way to check for visibility before changing a few of these in #14937 and didn't see examples recommending we also check with `expect`.

Reply:
Hm this says that it returns a promise that resovles to boolean. I think we still need to use `expect`?

Reply:
Hmm when I look at the docs like [here](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible), it looks like we don't need it 🤔 but, 🤷‍♀️